org.idempiere.test/src/org/idempiere/test/model/MTestUUTest.java

```diff
@@ -82,10 +82,20 @@ public void testInsertingTestUU() {
 	public void testDeletingTestUU() {
 		Properties ctx = Env.getCtx();
 		String trxName = getTrxName();
-		MTestUU testuu = new MTestUU(ctx, TestRecordInGardenWorld, trxName);
-		testuu.deleteEx(true);
-		MTestUU testuu2 = new MTestUU(ctx, TestRecordInGardenWorld, trxName);
-	    assertFalse(testuu2.get_UUID().equals(TestRecordInGardenWorld));
+
+		// deleting the TestRecordInGardenWorld is creating a deadlock with testReadingUpdatingTestUU in parallel execution
```

Comment:
This is not a suggestion but would to point out that the alternative solution for this is to add `@Execution(ExecutionMode.SAME_THREAD)` annotation to the MTestUUTest class.

Reply:
Thanks @hengsin - your suggestion sounds better.

I did a test and it worked fine, but this is sporadic, so let's push it and see if jenkins complains again.